client/src/sections/quests/Level8.tsx

```diff
@@ -8,7 +8,7 @@ import { atStep, Step, useQuestStep } from "../../hooks/useQuest";
 import { $item } from "../../util/makeValue";
 import { commaAnd, plural, truthy } from "../../util/text";
 
-const Level4: React.FC = () => {
+const Level8: React.FC = () => {
```

Comment:
bean level 8 looks good, i don't see any issues and the wording looks pretty good/sparse to me. my only thought is: do we want to recommend mountain man via cargo shorts here in the ore section, or do we just ignore it until cargo shorts tile? while guide/tourguide usually segregate iotm recommendations away from quest stuff, it doesn't -always-, and that is a place where i can't decide if we should match behavior or do otherwise. i don't see any other issues though, like this one a lot.

Reply:
awesome i will do this